lib/config/presets/internal/replacements.ts

```diff
@@ -0,0 +1,17 @@
+import type { Preset } from '../types';
+
+export const presets: Record<string, Preset> = {
+  replacements: {
+    packageRules: [
+      {
+        matchDatasources: ['npm'],
+        matchPackageNames: ['jade'],
+        replacementName: 'pug',
+        replacementVersion: '2.0.0',
+        commitMessageAction: 'Replace',
+        commitMessageExtra:
+          'with {{newName}} {{#if isMajor}}v{{{newMajor}}}{{else}}{{#if isSingleVersion}}v{{{newVersion}}}{{else}}{{{newValue}}}{{/if}}{{/if}}',
```

Comment:
Why do we need this in the preset? Can't these be the default values?

Reply:
Yes, I think you're right. I can define these in 'lib/config/definitions.ts`